packages/mobile/package.json

```diff
@@ -77,6 +77,8 @@
     "graphql": "^14.1.1",
     "hermes-engine": "0.4.2-rc1",
     "i18next": "^19.4.3",
+    "io-ts": "2.0.1",
+    "fp-ts": "^2.8.1",
```

Comment:
Can you use 2.1.1 to stay consistent with the other packages? Or upgrade the others.